src/Combobox/index.test.js

```diff
@@ -184,16 +184,16 @@ describe("Combobox", () => {
     expect(screen.getByDisplayValue("New York")).toBeInTheDocument();
   });
 
-  it("should not update input value in uncontrolled manner when `inputValue` is passed", () => {
+  it("should NOT update input value in UNcontrolled manner when `inputValue` is passed", () => {
     const value = "Initial value unhandled by parent state";
     render(
       <Combobox label={LABEL} inputValue={value}>
         {STATE_ITEMS}
-      </Combobox>
+      </Combobox>,
     );
     const input = screen.getByPlaceholderText(LABEL);
     expect(screen.getByDisplayValue(value)).toBeInTheDocument();
     fireEvent.change(input, { target: { value: "New value" } });
-    expect(screen.getByDisplayValue(value)).toBeInTheDocument();
+    expect(screen.getByDisplayValue(value)).not.toBeInTheDocument();
```

Comment:
This is a little embarrassing. 😳 
I wrote this combobox test to specifically check for the weird behavior this PR fixes, but my assertion was the opposite of what it should have been.